README.md

```diff
@@ -0,0 +1,289 @@
+|                                    |                   | [AddConfiguration](https://www.onvif.org/ver20/media/wsdl/media.wsdl#op.AddConfiguration)                                       | Object           |
+|                                    |                   | [RemoveConfiguration](https://www.onvif.org/ver20/media/wsdl/media.wsdl#op.RemoveConfiguration)                                 | Object           |
+| Analytics Module configuration     | Analytics         | [GetSupportedAnalyticsModules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetSupportedAnalyticsModules)       | Object           |
+|                                    |                   | [GetAnalyticsModules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetAnalyticsModules)                         | Object           |
+|                                    |                   | [CreateAnalyticsModules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.CreateAnalyticsModules)                   | Object           |
+|                                    |                   | [DeleteAnalyticsModules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.DeleteAnalyticsModules)                   | Object           |
+|                                    |                   | [GetAnalyticsModuleOptions](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetAnalyticsModuleOptions)             | Object           |
+|                                    |                   | [ModifyAnalyticsModules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.ModifyAnalyticsModules)                   | Object           |
+| Rule configuration                 | Analytics         | [GetSupportedRules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetSupportedRules)                             | Object           |
+|                                    |                   | [GetRules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetRules)                                               | Object           |
+|                                    |                   | [CreateRules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.CreateRules)                                         | Object           |
+|                                    |                   | [DeleteRules](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.DeleteRules)                                         | Object           |
+|                                    |                   | [GetRuleOptions](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.GetRuleOptions)                                   | Object           |
+|                                    |                   | [ModifyRule](https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.ModifyRule)                                           | Object           |
```

Comment:
Link should be https://www.onvif.org/ver20/analytics/wsdl/analytics.wsdl#op.ModifyRules

Reply:
Fixed the wrong link.